assets/blocks/course-outline/style.editor.scss

```
@@ -12,10 +12,9 @@ $dark-gray: #1a1d20;
 		}
 	}
 
-	.wp-block-sensei-lms-course-outline-module {
-		&__title-input {
-			background: none;
-		}
+	textarea.wp-block-sensei-lms-course-outline-module__title-input {
+		color: inherit;
+		background: none;
```

There is another issue related to the placeholder here. Maybe we could fix it adding it:

```suggestion
		background: none;
		
		&::placeholder {
			color: inherit;
			opacity: 0.5;
		}
```

It's messed by this file: `/wp-includes/css/dist/edit-post/style.css`

---

Nice one! Didn't notice that the default text was missing 😄